### PR TITLE
US001_initialization

### PR DESCRIPTION
**Pull request by kuzivka**
- Created project
- Deleted all unnecessary files
- Install some libraries (redux, MUI, date-fns)
- Added files and folders to form project structure
- Created task interface and state type

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```
Comment (Kost927) [inline .gitignore]: package-lock.json do not have to be in gitignore

File: src/App.tsx [hunk 2]
```diff
@@ -1,24 +1,15 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useStore } from 'react-redux';
+
 
 function App() {
+  const store = useStore();
+
+  console.log(store);
```
Comment (Kost927) [inline src/App.tsx]: remove console.log from the pull request please

File: src/actions/actionCreators.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { ADD_TODO } from './actions';
+
+// export const addTodo = (description, timestamp) => {
+//     return {
+//         type: ADD_TODO,
+//     payload: {
+//         description,
```
Comment (Kost927) [inline src/actions/actionCreators.ts]: 1) do not push commented code
2) configure your project to use the same style (single quotes or double quotes) coz somewhere I saw double quotes

File: src/type.d.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+interface ToDo {
+  id: number,
+  task: string,
+  createdAt: Date,
+  expiresAt: Date,
+  done: boolean,
+}
+
+type ToDoState = {
+  tasks: ToDo[]
```
Comment (Kost927) [inline src/type.d.ts:10]: to describe objects better to use interfaces

File: src/reducers/todoReducer.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+
+const initialState: ToDoState = {
+    tasks:[]
+}
+
+export const todoReducer = (state: ToDoState = initialState, action: any) => {
```
Comment (Kost927) [inline src/reducers/todoReducer.ts]: type any, there is no sense in using typescript with if you use any type

File: src/actions/actions.ts [hunk 6]
```diff
@@ -0,0 +1,3 @@
+export const ADD_TODO = 'ADD_TODO';
+export const EDIT_TODO = 'EDIT_TODO';
+export const DELETE_TODO = 'DELETE_TODO';
```
Comment (Yuliia143) [inline src/actions/actions.ts]: GitHub needs empty line at the end of each file you want to add

File: src/reducers/todoReducer.ts [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+    return state;
+};
```
Comment (Yuliia143) [inline src/reducers/todoReducer.ts]: GitHub needs empty line at the end of each file you want to add

**Pull request merged**